<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>paHMM-Tree by marbogusz</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">paHMM-Tree</h1>
      <h2 class="project-tagline">paHMM-Tree -  distance-based statistical phylogenetic tree estimation using pair hidden Markov models</h2>
      <a href="https://github.com/marbogusz/paHMM-Tree" class="btn">View on GitHub</a>
      <a href="https://github.com/marbogusz/paHMM-Tree/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/marbogusz/paHMM-Tree/tarball/master" class="btn">Download .tar.gz</a>
    </section>

    <section class="main-content">
      <h3>
<a id="welcome-to-pahmm-tree-webpage" class="anchor" href="#welcome-to-pahmm-tree-webpage" aria-hidden="true"><span class="octicon octicon-link"></span></a>Welcome to paHMM-Tree webpage.</h3>

<p>paHMM-Tree, pronounced as 'palm tree' is a phylogenetic tree inference tool that uses the principle of pairwise statistical alignment to construct a neighbour joining trees from raw sequences. </p>

<h3>
<a id="what-pahmm-tree-does" class="anchor" href="#what-pahmm-tree-does" aria-hidden="true"><span class="octicon octicon-link"></span></a>What paHMM-Tree does</h3>

<p>paHMM-Tree outputs phylogenetic trees in NEWICK format from raw sequence data in FASTA format. 
It does not require a prior sequence alignment. If you feed it with aligned sequenced, the gaps will be ignored. 
Being based on a pair hidden Markov model (pair-HMM) paHMM-Tree is able to integrate over all possible alignments
between all the pairs of sequences in the data set. By doing so it accounts for alignment uncertainty. </p>

<h3>
<a id="what-it-does-not-do" class="anchor" href="#what-it-does-not-do" aria-hidden="true"><span class="octicon octicon-link"></span></a>What it does not do</h3>

<p>paHMM-Tree does not align the sequences. In principle it could, but the whole idea is to avoid conditioning on a single multiple sequence alignment (MSA). Also constructing MSAs from pairwise alignments might not lead to reliable results.</p>

<h3>
<a id="documentation" class="anchor" href="#documentation" aria-hidden="true"><span class="octicon octicon-link"></span></a>Documentation</h3>

<h3>
<a id="support-or-contact" class="anchor" href="#support-or-contact" aria-hidden="true"><span class="octicon octicon-link"></span></a>Support or Contact</h3>

<p>Please report the issues and feature requests on GitHub. For any other requests contact Marcin Bogusz at <a href="mailto:Marcin.Bogusz@ebc.uu.se">Marcin.Bogusz@ebc.uu.se</a> </p>

      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/marbogusz/paHMM-Tree">paHMM-Tree</a> is maintained by <a href="https://github.com/marbogusz">marbogusz</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
